src/components/pages/realisations/CategoryFilters/RadioButton/index.js

```diff
@@ -31,20 +32,22 @@ const RadioButtonLabel = styled.label`
   text-transform: uppercase;
   font-size: 12px;
   line-height: 25px;
+  cursor: pointer;
 `;
 
-const RadioButton = ({ category, onChange, selectedValue }) => {
+const RadioButton = ({ category, onChange, selectedCategory }) => {
   return (
     <RadioButtonWrapper>
       <RadioButtonInput
         type="radio"
         id={category}
+        name="category"
         value={category}
-        checked={selectedValue === category}
+        defaultChecked={selectedCategory === category}
         onChange={onChange}
       />
 
-      <RadioButtonLabel for={category}>{category}</RadioButtonLabel>
+      <RadioButtonLabel htmlFor={category}>{category}</RadioButtonLabel>
```

Comment:
C'est quoi `htmlFor` ?

Reply:
htmlFor remplace for. j'avais un warning dans la console du DOM qui me demandait de faire ce changement.